WakaTime.py

```diff
@@ -120,6 +170,11 @@ def send(cls, heartbeat, extra_heartbeats=None):
             '--time', f'{heartbeat["timestamp"]:f}',
             '--plugin', ua,
         ]
+        blender_settings = bpy.context.preferences.addons[__name__].preferences
```

Comment:
Can't we just get the settings once and cache them?

Reply:
I am not sure, how costly the `bpy.context.preferences.addons[__name__].preferences` is, but how would the alternative look like?
Something like this?:
```
    global _blender_settings
    if _blender_settings == None:
        _blender_settings = bpy.context.preferences.addons[__name__].preferences
```

Reply:
Judging by the docs and examples there in, it should be too costly. So let's leave it as it is and see if we get any issues about performance.